lightning_server.cc

```diff
@@ -0,0 +1,64 @@
+#include "config_parser.h"
+#include "lightning_server.h"
+
+#include <array>
+#include <iostream>
+#include <string>
+
+// TODO: Refine debug output
+// TODO: Namespacing
+// TODO: Handle errors
+// TODO: Update integration test script
+const int MAX_REQ_SIZE = 8192;
```

Comment:
TODO: We'll want to figure out what real-world constraints/concerns govern this `MAX_REQ_SIZE`.